src/app/TideMatchSet.cpp

```diff
@@ -307,149 +307,162 @@ void TideMatchSet::report(
+      string flankingAAs, n_term, c_term;
+      getFlankingAAs(peptide, protein, pos, &n_term, &c_term);
+      flankingAAs = n_term + c_term;
+
+      // look for other locations
+      if (peptide->HasAuxLocationsIndex()) {
+         const pb::AuxLocation* aux = locations[peptide->AuxLocationsIndex()];
+         for (int j = 0; j < aux->location_size(); j++) {
+            const pb::Location& location = aux->location(j);
+              protein = proteins[location.protein_id()];
+              pos = location.pos();
+              proteinNames += "," + getProteinName(*protein, (!protein->has_target_pos()) ? pos : protein->target_pos());
+              getFlankingAAs(peptide, protein, pos, &n_term, &c_term);
+              flankingAAs += "," + n_term + c_term;
+         }
+      }
+
+      Crux::Peptide cruxPep = getCruxPeptide(peptide);
+      const SpScorer::SpScoreData* sp_data = sp_map ? &(sp_map->at(i).first) : NULL;
+
+      // FILE_COL, SCAN_COL, CHARGE_COL, SPECTRUM_PRECURSOR_MZ_COL, SPECTRUM_NEUTRAL_MASS_COL, PEPTIDE_MASS_COL, DELTA_CN_COL, DELTA_LCN_COL,
+      *file << spectrum_filename << '\t'
+           << spectrum->SpectrumNumber() << '\t'
+           << charge << '\t'
+           << StringUtils::ToString(spectrum->PrecursorMZ(), massPrecision) << '\t'
+           << StringUtils::ToString((spectrum->PrecursorMZ() - MASS_PROTON) * charge, massPrecision) << '\t'
+            << StringUtils::ToString(cruxPep.calcModifiedMass(), massPrecision) << '\t'
+            << delta_cn_map->at(i) << '\t'
+            << delta_lcn_map->at(i) << '\t';
+
+      if (sp_map) {
+         // SP_SCORE_COL, SP_RANK_COL, BY_IONS_MATCHED_COL, BY_IONS_TOTAL_COL
+         *file << StringUtils::ToString(sp_data->sp_score, precision) << '\t'
+              << sp_map->at(i).second << '\t'
+              << sp_data->matched_ions << '\t'
+              << sp_data->total_ions << '\t';
+      }
+
+      // XCORR_SCORE_COL, TAILOR_COL, XCORR_RANK_COL
+      *file << StringUtils::ToString(i->xcorr_score, precision, true) << '\t'
+             << StringUtils::ToString(i->tailor, precision, true) << '\t'
+             << rank << '\t';
+
+      // PRECURSOR_INTENSITY_RANK_M0_COL, PRECURSOR_INTENSITY_RANK_M1_COL, PRECURSOR_INTENSITY_RANK_M2_COL
+      // boost::tuple<double, double, double> intensity_tuple_old = intensity_map_old->at(i);
+      boost::tuple<double, double, double> intensity_tuple_new = intensity_map_new->at(i);
+      boost::tuple<double, double, double> logrank_tuple_new = logrank_map_new->at(i);
+      *file << StringUtils::ToString(intensity_tuple_new.get<0>()+intensity_tuple_new.get<1>()+intensity_tuple_new.get<2>(), precision, true) << '\t'
+           << StringUtils::ToString(intensity_tuple_new.get<0>(), precision, true) << '\t'
+           << StringUtils::ToString(logrank_tuple_new.get<0>()+logrank_tuple_new.get<1>()+logrank_tuple_new.get<2>(), precision, true) << '\t';
+
+      // RT_DIFF_COL
+      double predrt = 0.5;
```

Comment:
why is this set to 0.5?

Reply:
in case the predict rt is missing, we impute the default value as 0.5